openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMAlgorithms.java

```diff
@@ -35,11 +35,21 @@ public enum LightGBMAlgorithms implements MLAlgorithmEnum {
      * LightGBM binary classifier algorithm.
      */
     LIGHTGBM_BINARY_CLASSIFIER(createDescriptor(
-            "LightGBM Binary Classifier",
+            "LightGBM",
```

Comment:
I'm not sure we should remove binary classifier, we can't use this lgbm implementation for regression for instance.

Reply:
Let's revert the algorithm name change as it can possibly break existing models.